packages/kotti-ui/source/kotti-inline-edit-2/KtInlineEdit2.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+	<div :class="objectClass">
+		<label class="form-label" v-text="label" />
+		<div v-if="isEditing">
+			<component
+				:is="inputComponent"
+				class="form-input"
+				type="text"
+				:value="currentValue"
+				@change="handleChange"
+				@input="handleInput"
+				v-text="currentValue"
+			/>
+			<KtButtonGroup class="inline-edit__edit-buttons">
+				<KtButton icon="close" @click="handleDismiss" />
+				<KtButton icon="check" @click="handleConfirm" />
+			</KtButtonGroup>
+		</div>
+		<div
+			v-else
+			:class="representedValueClass"
+			@click.stop="isEditing = true"
+			v-text="representedValue"
+		/>
+	</div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, ref } from '@vue/composition-api'
+
+import { KtButton } from '../kotti-button'
+import { KtButtonGroup } from '../kotti-button-group'
+import { makeProps } from '../make-props'
+
+import { DEFAULT_POST_PARSER } from './types'
+import { KottiInlineEdit2 } from './types'
+
+export default defineComponent({
```

Comment:
Missing type annotation

```suggestion
export default defineComponent<KottiInlineEdit2.PropsInternal>({
```